app/src/main/java/io/github/wulkanowy/data/repositories/note/NoteRepository.kt

```diff
@@ -30,24 +30,24 @@ class NoteRepository @Inject constructor(
                             local.deleteNotes(old - new)
                             local.saveNotes((new - old)
                                 .onEach {
-                                    if (student.registrationDate <= it.date.atStartOfDay()) {
-                                        if (notify) it.isNotified = false
-                                        it.isRead = false
+                                    if (it.date.atStartOfDay() >= student.registrationDate) it.apply {
```

Comment:
Zmień na 
```suggestion
                                    if (it.date >= student.registrationDate.toLocalDate()) it.apply {
```

żeby uwagi z dnia rejestracji były oznaczone jako nieprzeczytane